catalog/MDCCatalog/MDCThemePickerViewController.swift

```diff
@@ -36,36 +47,37 @@ class MDCThemePickerViewController: UIViewController, UICollectionViewDataSource
   let titleColor = AppTheme.globalTheme.colorScheme.onSurfaceColor.withAlphaComponent(0.5)
   let titleFont = AppTheme.globalTheme.typographyScheme.button
   private let cellReuseIdentifier = "cell"
-  private let colorSchemeCells = [
-    (
-      mainColor: AppTheme.defaultTheme.colorScheme.primaryColor,
-      colorScheme: { return AppTheme.defaultTheme.colorScheme }
+  private let colorSchemeConfigurations = [
+    MDCColorThemeCellConfiguration(name: "Default",
+                                   mainColor: AppTheme.defaultTheme.colorScheme.primaryColor,
+                                   colorScheme: { return AppTheme.defaultTheme.colorScheme }()
     ),
-    (
-      mainColor: MDCPalette.blue.tint500,
-      colorScheme: { return createSchemeWithPalette(MDCPalette.blue) }
+    MDCColorThemeCellConfiguration(name: "Blue",
+                                   mainColor: MDCPalette.blue.tint500,
+                                   colorScheme: { return createSchemeWithPalette(MDCPalette.blue) }()
```

Comment:
Pretty sure you can change all of these to `colorScheme: createSchemeWithPalette(MDCPalette.blue)` etc. Not necessary to wrap the function call in a closure and execute that

Reply:
Oh, I thought it was going to be lazy and not evaluate the closure until it was referenced?

Reply:
Except that my code is no longer lazy!  Spoke with @yarneo and we'll decide whether to keep the closures or do proper properties.

Reply:
So the thought process behind making it lazy is that we won't have to initialize X many color schemes but rather only the color scheme you would need. Does that make sense? I felt like the scheme creation so many times was unneeded and can be heavy.

Reply:
I understood the reasons behind it, I just didn't know how much we care to keep it around.  Since I believe you were the original author, I thought you might have an opinion one way or the other.